src/pages/Orgs/pages/OrgsId/pages/OrgCheckProof/components/CheckProofHead.tsx

```diff
@@ -10,7 +10,7 @@ export default function CheckProofHead() {
 
   return (
     <Box position={'relative'} mx={-8} px={6} pb={6} borderBottom={1} borderColor={palette.divider}>
-      <Box position={'absolute'} top={spacing(2)} left={spacing(6)}>
+      <Box position={'absolute'} top={spacing(1)} left={0}>
```

Comment:
does "top" property ain't use `spacing` as default?

Reply:
no, it's only used for margin/padding properties